pkg/cache/nodes.go

```diff
@@ -217,7 +217,7 @@ func (nc *schedulerNodes) schedulerNodeEventHandler() func(obj interface{}) {
 }
 
 func hasReadyCondition(node *v1.Node) bool {
-	if node != nil && node.Status.String() != "nil" {
+	if node != nil {
```

Comment:
can node.Status be a nil? do we need
```
node.Status != nil?
```

Reply:
No it cannot be nil as it is not a pointer it is a NodeStatus struct.